docs/en/stack/ml/anomaly-detection/ml-troubleshooting.asciidoc

```diff
@@ -8,6 +8,42 @@
 Use the information in this section to troubleshoot common problems and known
 issues.
 
+[discrete]
+[[ml-avoid-upgrade-closures]]
+== Avoiding spurious {anomaly-job} closures on upgrade
+
+When you perform a rolling upgrade to _or_ from versions
+7.14.0 or 7.14.1 you may find that {anomaly-jobs} that
+were `opened` during the upgrade incorrectly end up
+`closed` after the upgrade.
+
+*Symptoms:*
+
+* Some (but not necessarily all) {anomaly-jobs} that were
+  in the `opened` state before the upgrade show as `closed`
+  after upgrade.
```

Comment:
> after upgrade.

The issue also mentioned "Job ended up blocked with revert". I presume that's the blocked property mentioned in 
https://www.elastic.co/guide/en/elasticsearch/reference/7.14/ml-get-job.html. Should it be mentioned as a symptom of the problem here too?

Reply:
Good point, it should.

And also I missed:

> The only way to recover from this state is to manually revert to the model snapshot with ID `empty`

out of the remediation steps.

That makes this problem _far_ worse if it really is the only way to recover.  I need to check if reverting to the same model snapshot that the block mentions is an alternative, and add one or the other to the remediation steps.

Reply:
Thanks! For now I've added a blurb about the "blocked" property in c3fd7b9d1b4d2e4ff62e2ba4edaa71555003565e